packages/styles/scss/patterns/blocks/contentgroupcards/_contentgroupcards-item.scss

```diff
@@ -5,13 +5,12 @@
 // LICENSE file in the root directory of this source tree.
 //
 
-@import '../../../components/link-with-icon/link-with-icon';
-
 @mixin content-group-cards-item {
   .#{$prefix}--content-group-cards-item {
+    h3,
     p {
-      margin-bottom: $carbon--layout-03;
-      margin-top: $carbon--layout-03;
+      width: 90%;
+      max-width: carbon--mini-units(80);
```

Comment:
I think we should create a mixin for this, and attack the root sub-patterns. Not the decorators. Also, are there no classnames we can use to style with instead of the `html`?

Reply:
I've updated the selectors so they use the `Card` classes.